sdk/src/test/java/com/mapbox/maps/MapboxMapTest.kt

```diff
@@ -31,28 +37,36 @@ class MapboxMapTest {
 
   @Test
   fun loadStyleUri() {
+    Shadows.shadowOf(Looper.getMainLooper()).pause()
     mapboxMap.loadStyleUri("foo")
+    Shadows.shadowOf(Looper.getMainLooper()).idle()
```

Comment:
@kiryldz just fyi, I have been using the [this](https://github.com/mapbox/mapbox-maps-android/blob/a757ddb3deaaed600b69ab9b61114b53551b992a/sdk/src/test/java/com/mapbox/maps/NativeObserverTest.kt#L27) to mock `handler.post { }`. This way we can avoid duplicated `pause` and `idle` code in the tests.